graph-server/src/index.js

```diff
@@ -43,12 +43,24 @@ app.get(`${PATH_CONTEXT}/monitoring-location/:siteID/`, cache({ttl: CACHE_TIMEOU
             }
         }
     },
+    width: {
+        in: ['query'],
+        errorMessage: 'width should be an integer',
+        isInt: true,
```

Comment:
Wondering it's easy to add options to this... if so would it be worthwhile to add `{gt: 0}` as a hedge against negative integers? That said, I think the probability of someone doing that is quite low. :thinking:

Reply:
Yes. I think that's a good idea.

Reply:
Decided to set min and max to 300 and 1200, respectively. Thanks for the suggestion!